app/views/pages/home.html.erb

```diff
@@ -1,3 +1,10 @@
-<div class="container">
-  <%= render "pages/partials/membership" %>
```

Comment:
It just looks like there's a repeat here. One should be deleted, right?

Reply:
The red lines are what is deleted and the green lines are what's been added
So this looks like this in reality :
```
<div id="home">
  <div class="container">
    <%= render "pages/partials/membership" %>

    <section id="planning">
      <h2>Planning</h2>
      <%= render "pages/partials/calendar" %>
    </section>
  </div>
</div>
```